sources/packages/api/src/services/application/application.service.ts

```diff
@@ -1442,4 +1443,33 @@ export class ApplicationService extends RecordDataModelService<Application> {
       .setParameter("isApplicationArchived", true)
       .execute();
   }
+
+  /**
+   * Get offering request change for a particular application id.
+   * @param applicationId
+   * @returns Offering id and submitted date.
```

Comment:
pls refactor the comment

Reply:
it's not returning the submitted date. 
![image](https://user-images.githubusercontent.com/77353155/186499711-c0d8a316-54d1-4be5-9fe8-8032bde19796.png). may you can update it something like `@returns application assessment and offering details`